actors/builtin/market/market_test.go

```diff
@@ -2017,19 +2042,41 @@ func (h *marketActorTestHarness) withdrawClientBalance(rt *mock.Runtime, client
 	rt.Verify()
 }
 
-func (h *marketActorTestHarness) cronTickNoChangeBalances(rt *mock.Runtime, client, provider address.Address) {
-	// fetch current client and provider escrow balances
-	cLocked := h.getLockedBalance(rt, client)
-	cEscrow := h.getEscrowBalance(rt, client)
-	pLocked := h.getLockedBalance(rt, provider)
-	pEscrow := h.getEscrowBalance(rt, provider)
+func (h *marketActorTestHarness) cronTickNoChangeBalances(rt *mock.Runtime, client, provider address.Address, clientRemoved, providerRemoved bool) {
```

Comment:
Since this is `false, false` at all but one call site, it might be less noisy to have this method ignore removals, and check for removal explicitly at the one site where it's expected.

Change getEscrowBalance to return zero for missing accounts, if that makes it easier.

Reply:
This is done.